lib/tasks/docker.rake

```diff
@@ -214,6 +218,8 @@ task "docker:test" do
         )
 
       unless ENV["JS_ONLY"]
+        @good &&= run_or_fail("bin/ember-cli --build") if ENV["RUN_SYSTEM_TESTS"]
```

Comment:
This is a bug, we need to run `bin/ember-cli` build as long as `RUN_SYSTEM_TEST` is present. Otherwise, plugin system tests will fail if `SKIP_CORE` is used.